code/modules/maps/planet_types/lore/tauceti.dm

```diff
@@ -0,0 +1,112 @@
+// --------------------------------- Caprice
+
+/obj/effect/overmap/visitable/sector/exoplanet/lava/caprice
+	name = "Caprice"
+	desc = "A scorching-hot volcanic planet close to Tau Ceti."
+	charted = "Charted 2147CE, Sol Alliance Department of Colonization."
+	icon_state = "globe1"
+	color = "#cf1020"
+	generated_name = FALSE
+	ring_chance = 0
+
+/obj/effect/overmap/visitable/sector/exoplanet/lava/caprice/update_icon()
+	return
+
+/obj/effect/overmap/visitable/sector/exoplanet/lava/caprice/generate_ground_survey_result()
+	ground_survey_result = "" // so it does not get randomly generated survey results
```

Comment:
All new planets should have survey results defined for survey probes.

Reply:
I can help with these if it's not clear what should be put there.

Reply:
this was not resolved - now they just use the randomly generated survey results, different every round